core/rfb.js

```diff
@@ -790,8 +797,15 @@ export default class RFB extends EventTargetMixin {
 
         if (this._viewOnly) { return; } // View only, skip mouse events
 
-        if (this._rfb_connection_state !== 'connected') { return; }
-        RFB.messages.pointerEvent(this._sock, this._display.absX(x), this._display.absY(y), this._mouse_buttonMask);
+        // Limit mouse move events to one every MOUSE_MOVE_DELAY mS
+        clearTimeout(this.timerID);     // Remove old one
+        this.newMouseMoveTime = Date.now();
+        if ( this.newMouseMoveTime < this.oldMouseMoveTime + MOUSE_MOVE_DELAY ) {
+            this.timerID = setTimeout(this._setMousePosition, MOUSE_MOVE_DELAY, x, y);  // Set last mouse position after timeout
```

Comment:
I don't think this works properly. `this` would be incorrectly set when this timeout fires. Have you tested it?

On that note, some unit tests would be nice for this.

Reply:
fixed bind()

and added unit tests in a672168d4dac5735b40cfe618a47f76b69c85882